### DISCUSSION TITLE
Confidence difference in yolov8 vs yolov4

### DISCUSSION THREAD

**Discussion by IgorGoldberg** [ultralytics/ultralytics#14588] [Q&A]
Hi. 
Recently I moved one of my studies from yolo4 (actually it is Nvidia Tao yolov4) to yolov8 and I observe a strange phenomena where during an inference in yolo v8 the confidences are significantly lower on same inference images comparing to the yolov4 confidences. 
The problem is that I have a thresholds  based postprocessing, and I am not quite sure how to adjust the thresolds to fit yolov8 results
Thus the solution may be in 2 directions:
1) findout why we get a low probabilities and fix this issue
2) as an alternative, maybe someone could give me a clue whether there pissibly exists some translation formula where the input will be yolov4 confidence and the output is yolov8. Of course I could gather eniugh data and try to figure out the translation rule out of it, but I thought that maybe its is a known issue that someone met before.

**Comment by glenn-jocher** [comment]
@IgorGoldberg hello,

The difference in confidence scores between YOLOv4 and YOLOv8 can be attributed to variations in model architecture, training data, and optimization techniques. These differences can affect how each model interprets and scores detections.

To address this:
1. Ensure you are using the latest version of YOLOv8, as updates may include improvements that could affect confidence scores.
2. Adjust your post-processing thresholds to better align with YOLOv8's confidence outputs. This might involve some trial and error to find the optimal settings.
3. There isn't a direct translation formula between YOLOv4 and YOLOv8 confidences due to their inherent differences. However, you can empirically derive a mapping by analyzing a set of inference results from both models and adjusting your thresholds accordingly.

If the issue persists, please provide more details or examples, and we can further investigate.
